seqr/views/apis/data_manager_api.py

```diff
@@ -387,14 +387,75 @@ def load_rna_seq_sample_data(request, sample_guid):
         data_by_gene = json.loads(row.split('\t\t')[1])
 
     model_cls = RNA_DATA_TYPE_CONFIGS[data_type]['model_class']
-    models = model_cls.objects.bulk_create([model_cls(sample=sample, **data) for data in data_by_gene.values()])
-    logger.info(f'create {len(models)} {model_cls.__name__}', request.user, db_update={
-        'dbEntity': model_cls.__name__, 'numEntities': len(models), 'parentEntityIds': [sample_guid], 'updateType': 'bulk_create',
```

Comment:
I think ultimately we do want to maintain the `parentEntityIds` part of this log. You can add it back to the helper by either making `parent_ids ` an argument to `bulk_<create/delete>` and/or by adding a PARENT_FIELD property to anything that inherits from `BulkOperationBase` and the updating the logging helper to compute it:
```
def log_model_no_guid_bulk_update(logger, models, user, update_type):
    db_entity = type(models[0]).__name__
    prefetch_related_objects(models, db_entity.PARENT_FIELD)
    parent_ids = {getattr(model, db_entity.PARENT_FIELD).guid for model in models}
    ...
```

Reply:
The `PARENT_FIELD` field sounds better to me.